### PR TITLE
Enhance logging test

### PR DESCRIPTION
**Pull request by shorim**
**Description**

Changes proposed in this pull request:

- Increased the time interval between retrials of querying loki to avoid overwhelming loki server
- Query logs starting only from current time, so that the execution of query would be faster on loki side
- use GO HTTP client instead of executing curl commands

### CODE REVIEW

File: tests/end-to-end/upgrade/pkg/tests/logging/pkg/logstream/logstream.go [hunk 1]
```diff
@@ -103,3 +103,28 @@ func Cleanup(namespace string, coreInterface kubernetes.Interface) error {
...
 	return nil
 }
+
+func doGet(url string, authHeader string) (string, error) {
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return "", errors.Wrap(err, "cannot create a new HTTP request")
+	}
+	req.Header.Add("Authorization", authHeader)
+	client := &http.Client{
```
Comment (piotrmiskiewicz) [inline tests/end-to-end/upgrade/pkg/tests/logging/pkg/logstream/logstream.go:113]: It is not a good idea to create a new client for every call. Can you just share it between calls?

Comment (shorim) [reply tests/end-to-end/upgrade/pkg/tests/logging/pkg/logstream/logstream.go]: yes, I agree it is better to share the client.
The changes are pushed now

**Pull request merged**